### PR TITLE
Update oscenter docs README

### PR DESCRIPTION
**Pull request by ScorpioCPH**
@pendoragon @ddysher PTAL.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,9 @@
-# oscenter-docs
\ No newline at end of file
+# oss-docs
+
+Welcome to Caicloud OSS Documentation Repository!
+
+This is a group of Caicloud which focus on OSS community's projects (e.g. [Kubernetes](https://github.com/kubernetes/kubernetes), [TensorFlow](https://github.com/tensorflow/tensorflow)).
```
Comment (pendoragon) [inline README.md]: s/focus/focuses/

Comment (ddysher) [reply README.md]: How about `This is a group of Caiclouders focusing on blablabla`?

**Comment by pendoragon** [comment]
@ddysher to confirm if README captures everything this repo is for

File: README.md (hunk 1 above)
Comment (ScorpioCPH) [reply README.md]: sgtm.

**Comment by ddysher** [comment]
@pendoragon LGTM

**Pull request merged**